src/App/Fossa/ListTargets.hs

```diff
@@ -81,6 +81,7 @@ listTargetsMain ListTargetsConfig{..} = do
     . withTaskPool capabilities updateProgress
     . runAtomicCounter
     . runReader experimental
+    . runReader (mempty :: MavenScopeFilters)
```

Comment:
[Optional, Prefer] comment on why we do not care about scope filters for list-targets